msg/airspeed_validated.msg

```diff
@@ -0,0 +1,15 @@
+uint64 timestamp			# time since system start (microseconds)
+
+float32 indicated_airspeed_m_s		# indicated airspeed in m/s (IAS)
+
+float32 equivalent_airspeed_m_s     # equivalent airspeed in m/s (accounts for instrumentation errors) (EAS)
+
+float32 true_airspeed_m_s		# true filtered airspeed in m/s (TAS)
+
+float32 ground_minus_wind # TAS calculated from groundspeed - windspeed
+
+bool airspeed_sensor_measurement_valid # true if airspeed measurement from (one or multiple) pitot is valid
+
+bool airspeed_valid # true if airspeed_sensor_measurement_valid or groundspeed-windspeed is valid
```

Comment:
you mean setting all 4 airspeeds (IAS, EAS, TAS, ground-wind) to NaN instead of having the airspeed_valid flag? I think I'd prefer to have it as it is now (mainly for easier debugging, plus maybe some people want to use the airspeed_validated even when the GPS is declared faulty). What's your view on this @RomanBapst ?

Reply:
@bkueng done